src/main/resources/application.yml

```diff
@@ -64,3 +66,6 @@ rest:
 uploads:
   ingestion-flow-file:
     valid-extension: "\${INGESTION_FLOW_FILE_VALID_EXTENSION:.zip}"
+
+app:
+  fileEncryptPassword: "\${FILE_ENCRYPT_PASSWORD:ENCR_PSW}"
```

Comment:
this variable should be set inside the helm project, it's a secret, thus it should be documented in:
https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1190330377/PU+-+Secret+manuali

And its value, for each environment, should be handled on the project p4pa-infra